regression/pages/lms/course_page_lms.py

```diff
@@ -31,4 +31,5 @@ def get_pages_name(self):
         Returns
             list: A list of names of all pages
         """
-        return self.q(css='.tabs.course-tabs .tab').text
+        tab_pages = self.q(css='.tabs.course-tabs .tab').text
+        return [text.replace('\n, current location', "") for text in tab_pages]
```

Comment:
Could you explain why we're adding this text? Wouldn't the index in the list be good enough?